lease/leasehttp/http_test.go

```diff
@@ -76,3 +122,32 @@ func TestTimeToLiveHTTP(t *testing.T) {
 		t.Fatalf("granted TTL expected 5, got %d", resp.LeaseTimeToLiveResponse.GrantedTTL)
 	}
 }
+
+func TestTimeToLiveHTTPTimeout(t *testing.T) {
```

Comment:
this is very similar to `TestRenewHTTPTimeout` both should probably call into a common function that has Renew/TimeToLive passed in